lib/ember-app.js

```diff
@@ -153,7 +153,13 @@ function serializeHTML(doc, rootElement) {
 function buildWhitelistedRequire(whitelist, distPath) {
   return function(moduleName) {
     if (whitelist.indexOf(moduleName) > -1) {
-      return require(path.join(distPath, 'node_modules', moduleName));
+      try {
+        // Attempt to see if it's a built-in module
+        return require(moduleName);
+      } catch (e) {
+        // See if it's an npm module, since it wasn't built in
+        return require(path.join(distPath, 'node_modules', moduleName));
```

Comment:
Chatted with @danmcclain in Slack, I think he is going to work on removing the `try` / `catch` if we can...